doc/source/user_guide/groupby.rst

```diff
@@ -1303,16 +1307,16 @@ column ``B`` because it is not numeric. We refer to these non-numeric columns as
    df.groupby("A").std(numeric_only=True)
 
 Note that ``df.groupby('A').colname.std().`` is more efficient than
-``df.groupby('A').std().colname``, so if the result of an aggregation function
-is only interesting over one column (here ``colname``), it may be filtered
+``df.groupby('A').std().colname``. So if the result of an aggregation function
+is only needed over one column (here ``colname``), it may be filtered
 *before* applying the aggregation function.
 
 .. note::
-   Any object column, also if it contains numerical values such as ``Decimal``
-   objects, is considered as a "nuisance" column. They are excluded from
-   aggregate functions automatically in groupby.
+   If an object column includes numerical values such as ``Decimal``
+   objects, it is considered a "nuisance" column. They are automatically
```

Comment:
Agreed this can be phrased better, but I believe the change here is incorrect - it states that a nuisance column must contain numerical values. Any object column is consider a nuisance column. I'd suggest "Any object column, even if it contains..."

Reply:
This note in general is out of date - `numeric_only` now defaults to False and so they will no longer be automatically excluded.

Reply:
I will change to your suggestion and remove the out of date note.